.github/.codespellignore

```diff
@@ -0,0 +1,6 @@
+cips
+pullrequest
+keypair
+stStr
+pastTime
+everytime
```

Comment:
proper english says this should be two words (https://www.grammarly.com/blog/everytime-every-time/#:~:text=Everytime%20should%20be%20written%20as,considered%20an%20acceptable%20compound%20word.)

```suggestion
```